src/freertos_drivers/st/Stm32Can.cxx

```diff
@@ -123,6 +130,19 @@ Stm32Can::Stm32Can(const char *name)
 #endif
```

Comment:
please set the interrupt priority on the third interrrupt as well.

Reply:
Fixed.